docs/pages/reference/backends.mdx

```diff
@@ -946,7 +946,32 @@ Config option priority is applied in the following order:
   are not supported in certain regions or environments or are only supported in GovCloud.
 </Admonition>
 
-## Athena
+## Athena (Preview)
+
+<Details
+  title="Version warning"
+  opened={true}
+  min="14.0"
+>
+  Athena audit back-end is available starting from Teleport `14.0`.
+</Details>
+
+<Admonition type="warning" title="Preview">
+  Athena audit back-end is currently in Preview.
+</Admonition>
```

Comment:
```suggestion
The Athena audit log back-end is available starting from Teleport v14.0, and is currently in preview.
```

There are a lot of boxes in a short span of content, which becomes pretty distracting. I've suggested condensing this content into regular body text.